.github/workflows/build_and_test.yml

```diff
@@ -366,7 +366,7 @@ jobs:
         #   See also https://github.com/sphinx-doc/sphinx/issues/7551.
         # Jinja2 3.0.0+ causes error when building with Sphinx.
         #   See also https://issues.apache.org/jira/browse/SPARK-35375.
-        python3.6 -m pip install flake8 pydata_sphinx_theme mypy numpydoc 'jinja2<3.0.0'
+        python3.6 -m pip install flake8 pydata_sphinx_theme mypy numpydoc black 'jinja2<3.0.0'
```

Comment:
Out of curiosity, do we run `black` with `python3.9`? Does the container have `black` already?

Reply:
Ah the linter workflow doesn't use the custom image yet.

Reply:
Makes sense, thanks!